src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalCatchCheck.java

```diff
@@ -56,7 +56,82 @@
  * <pre>
  * &lt;module name=&quot;IllegalCatch&quot;/&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
```

Comment:
Please make it single line
` <p>Example:</p>`

Reply:
done